server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -537,6 +562,11 @@ public void execute(final String[] args) {
       if (success && opts.addVolumes) {
         success = addVolumes(fs, initConfig, serverDirs);
       }
+      if (success && opts.resourceGroups != null) {
+        try (var zk = new ZooSession(getClass().getSimpleName(), siteConfig)) {
+          success = addResourceGroups(zk.asReaderWriter(), opts.resourceGroups);
```

Comment:
This is not a chrooted ZooSession. The paths are going to be wrong. This should probably use a ServerContext (see the `resetSecurity` method for a similar requirement and similar validation checks to ensure that the system, especially ZK, is already initialized).

Reply:
Fixed in 63fbabf